docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1238,72 +1283,70 @@ quarkus tls lets-encrypt issue-certificate \
++
+[source,shell]
 ----
 quarkus tls lets-encrypt renew-certificate \
   --domain=<domain-dns-name> <1>
 ----
 <1> Set your domain name.
-
-During this command, TLS registry CLI reads a Let's Encrypt account information recorded during the <<lets-encrypt-issue-certificate>> step, issues a Let's Encrypt certificate request, and communicates with a Quarkus application to have ACME challenges resolved.
-
++
+During this command, TLS registry CLI reads a Let's Encrypt account information recorded during the <<lets-encrypt-issue-certificate,Issue certificates with Let's Encrypt>> step, issues a Let's Encrypt certificate request, and communicates with a Quarkus application to have ACME challenges resolved.
++
 Once the Let's Encrypt certificate chain and private key have been successfully renewed, they are converted to PEM format and copied to your application's `.letsencrypt` folder.
 The TLS registry is notified when a new certificate and private key are ready, and it automatically reloads them.
 
-[[lets-encrypt-ngrok]]
-=== Use ngrok for testing
-
+. [[lets-encrypt-ngrok]] Testing with ngrok:
++
 https://ngrok.com/[ngrok] can be used to provide a secure HTTPS tunnel to your application running on localhost, and make it easy to test HTTPS based applications.
-
++
 ngrok provides a simplified way of getting started with the Quarkus Let's Encrypt ACME feature.
 
-The first thing you have to do with ngrok is to ask it to reserve a domain.
+.. Ask ngrok to reserve a domain:
++
 You can use https://github.com/quarkiverse/quarkus-ngrok[Quarkiverse ngrok] in dev mode or reserve it directly in the ngrok dashboard.
-
 Unfortunately, you cannot use your ngrok domain to test the Quarkus Let's Encrypt ACME feature immediately.
 This is because ngrok itself uses Let's Encrypt and intercepts ACME challenges that are meant to be handled by the Quarkus application instead.
 
-Therefore, remove the ngrok Let's Encrypt certificate policy from your ngrok domain:
-
-[source, shell]
+.. Therefore, remove the ngrok Let's Encrypt certificate policy from your ngrok domain:
++
+[source,shell]
 ----
 ngrok api --api-key <YOUR-API-KEY> reserved-domains delete-certificate-management-policy <YOUR-RESERVED-DOMAIN-ID>
 ----
-
++
 `YOUR-RESERVED-DOMAIN-ID` is your reserved domain's id which starts from `rd_`, you can find it in the https://dashboard.ngrok.com/cloud-edge/domains[ngrok dashboard domains section].
 
-Now, because ngrok only forwards ACME challenges over HTTP, start ngrok as follows:
-
-[source, shell]
+.. Now, because ngrok only forwards ACME challenges over HTTP, start ngrok as follows:
++
+[source,shell]
 ----
 ngrok http --domain <YOUR-NGROK-DOMAIN> 8080 --scheme http <1>
 ----
 <1> `8080` is the localhost HTTP port your application is listening on.
 
-You can now test the Quarkus Let's Encrypt ACME feature from your local machine.
+.. Test the Quarkus Let's Encrypt ACME feature from your local machine.
```

Comment:
@sberyozkin Hello Sergey! Do you think we need to add some more information here? Some more descriptions or commands/snippets?

Reply:
HI @MichalMaler I'm not sure replacing `You can now test ...` which finalizes what has been described above with `.. Test...` sounds right, it becomes somehow disconnected...
That command is all what is needed for Ngrok to support Let's Encrypt ACME challenges correctly. FYI, @cescoffier added another clarification here explaining what exactly NGrok does in this case.